specutils/spectra/spectral_coordinate.py

```diff
@@ -0,0 +1,651 @@
+import warnings
+
+import numpy as np
+from collections import namedtuple
+
+import astropy.units as u
+from astropy.constants import c
+from astropy.coordinates import SkyCoord, ICRS, Distance, GCRS, CartesianRepresentation, CartesianDifferential, SphericalDifferential, SphericalRepresentation
+from astropy.coordinates.baseframe import BaseCoordinateFrame, FrameMeta
+from astropy.utils.exceptions import AstropyUserWarning
+
+DOPPLER_CONVENTIONS = {
+    'radio': u.doppler_radio,
+    'optical': u.doppler_optical,
+    'relativistic': u.doppler_relativistic
+}
+
+RV_RS_EQUIV = [(u.cm / u.s, u.Unit(''),
+                lambda x: np.sqrt((1 + x/c.cgs.value)/(1 - x/c.cgs.value)) - 1,
+                lambda x: ((x + 1) ** 2 - 1) / ((x + 1) ** 2 + 1) * c.cgs.value)]
```

Comment:
This doesn't seem right to me (and I've coded this into the tests I pushed up) - the RV to redshift conversion is much simpler: v = c*z .  This looks like the relativistic doppler convention which is *not* the one used in the context of the normal definition of redshift.  So I think you just want this:


```suggestion
RV_RS_EQUIV = [(u.cm / u.s, u.Unit(''),
                lambda x: x/c.cgs.value,
                lambda x: x*c.cgs.value)]
```